queries/subsquid/general/collectionForMint.graphql

```diff
@@ -9,5 +9,8 @@ query collectionForMint($account: String!) {
     max
     minted: nftCount
```

Comment:
using minted does not work?

Reply:
it doesn't for all cases, if the collection has NFTs with non sequential `sn`(like we do with drops) you could end up trying to mint a used `sn`